lib/compress/zstd_compress.c

```diff
@@ -6527,3 +6659,19 @@ ZSTD_parameters ZSTD_getParams(int compressionLevel, unsigned long long srcSizeH
     if (srcSizeHint == 0) srcSizeHint = ZSTD_CONTENTSIZE_UNKNOWN;
     return ZSTD_getParams_internal(compressionLevel, srcSizeHint, dictSize, ZSTD_cpm_unknown);
 }
+
+void ZSTD_refExternalMatchFinder(
```

Comment:
It seems to me a regsiter function for extenralMatherFinder, Shall we call it ZSTD_regExternalMatchFinder?

Reply:
Sounds good to me, will change to ZSTD_registerExternalMatchFinder()